packages/terra-consumer-nav/src/Nav.jsx

```diff
@@ -1,31 +1,121 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import 'terra-base/lib/baseStyles';
+import QuickLinks from './components/quick-links/QuickLinks';
+import NavItems from './components/nav-items/NavItems';
 import styles from './Nav.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
- /*
```

Comment:
The props-table component uses react-docgen to generate the `props` table for each component. If you want to use that, the comments above each prop are used as a description. 

Here is an example:
https://github.com/cerner/terra-core/blob/master/packages/terra-arrange/src/Arrange.jsx#L11-L13